docs/html/user_guide.rst

```diff
@@ -94,10 +94,11 @@ pip can be configured to connect through a proxy server in various ways:
 * using the ``--proxy`` command-line option to specify a proxy in the form
   ``scheme://[user:passwd@]proxy.server:port``
 * using ``proxy`` in a :ref:`config-file`
+* by setting the environment-variable ``pip_proxy``
 * by setting the standard environment-variables ``http_proxy``, ``https_proxy``
-  and ``no_proxy``.
```

Comment:
Let's add a full stop to the first case, rather than removing it from the other 2.